lib/lib_rss.php

```diff
@@ -398,11 +398,12 @@ function is_referer_from_same_domain() {
 function check_install_php() {
 	$pdo_mysql = extension_loaded('pdo_mysql');
 	$pdo_sqlite = extension_loaded('pdo_sqlite');
+	$pdo_pgsql = extension_loaded('pdo_pgsql');
```

Comment:
Is this order (also below) the most sensible? With only three I suppose it doesn't matter but I'd go for alphabetic. :-)